torchdynamo/eval_frame.py

```diff
@@ -256,50 +268,100 @@ def dynamo_normalization_capturing_compiler(
         gm: torch.fx.GraphModule, example_inputs
     ):
         nonlocal graph
+        nonlocal compiler_captured_inputs
+
         assert graph is None, "whole graph export entails exactly one graph"
         graph = gm
-        return gm.forward
+        compiler_captured_inputs = example_inputs
+
+        def result_capturing_wrapper(*graph_inputs):
+            nonlocal graph_result_export
+            graph_result_export = graph(*graph_inputs)
```

Comment:
I'd expect us to need to be capturing the inputs too, for matching them.  id/is matching might not work otherwise.

Reply:
We do, L275 right above, but I agree, putting in here might read cleaner.